src/AutorestSwift/JsonRpc/AutorestPlugin.swift

```diff
@@ -98,7 +98,16 @@ class AutorestPlugin {
                 .success(.list([.string("swift")]))
             )
         case "process":
-            plugin.sessionId = params.asList?.last?.asString ?? ""
+            guard let sessionId = params.asList?.last?.asString else {
+                logger.log("Process request get invalid session id")
+                callback(.failure(RPCError(
+                    kind: .invalidParams,
+                    description: "Process request get invalid session id"
+                )))
+                return
+            }
+
+            plugin.sessionId = sessionId
```

Comment:
Since this is not a "production level" tool, we could also just declare `var sessionId: String!`.  Then you could keep the original, more concise usage.

```Swift
plugin.sessionId = params.asList?.last?.asString

print(plugin.sessionId) // automatically unwrapped
```